llvm/test/CodeGen/AMDGPU/fmin_legacy.f16.ll

```diff
@@ -114,8 +114,10 @@ define <2 x half> @test_fmin_legacy_ule_v2f16(<2 x half> %a, <2 x half> %b) #0 {
 ; VI-SAFE-LABEL: test_fmin_legacy_ule_v2f16:
 ; VI-SAFE:       ; %bb.0:
 ; VI-SAFE-NEXT:    s_waitcnt vmcnt(0) expcnt(0) lgkmcnt(0)
-; VI-SAFE-NEXT:    v_cmp_ngt_f16 vcc, v0, v1 src0_sel:WORD_1 src1_sel:WORD_1
-; VI-SAFE-NEXT:    v_cndmask_b32_sdwa v2, v1, v0, vcc dst_sel:WORD_1 dst_unused:UNUSED_PAD src0_sel:WORD_1 src1_sel:WORD_1
+; VI-SAFE-NEXT:    v_lshrrev_b32_e32 v2, 16, v1
```

Comment:
@arsenm This is an effect of always using the copy from the carry-in operand to VCC.